minecraft_servers/mypvp/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+  "server_name": "mypvp",
+  "nice_name": "MyPvP",
+  "direct_ip": "mypvp.me",
+  "server_wildcards": [
+    "%.mypvp.me",
+    "%.mypvp.eu",
+    "%.mypvp.us",
+    "%.mypvp.org",
+	"%.mypvp.net",
+	"%.mypvp.cloud",
+	"%.besterserver.de",
+	"%.minecraft.ms",
+	"%.minekube.eu",
+	"%.server.gratis",
+	"%.userserver.net"
+  ],
+  "social": {
+    "web": "https://mypvp.me",
+    "twitter": "MyPvPme",
+	"youtube": "https://www.youtube.com/channel/UCq2tbnNIT9ANK91uFExFC3g",
+    "discord": "https://discord.com/invite/uPXmSs6"
```

Comment:
Please re-format the part here. -> [Example manifest](https://github.com/LabyMod/server-media#example-manifestjson)